source/js/iframeAcceptance.js

```diff
@@ -32,7 +32,8 @@ class IframeAcceptance {
             [...document.querySelectorAll('iframe')].forEach(iframe => {
                 const wrapper = document.createElement('div');
                 const isVideo = iframe.parentElement.classList.contains("embed");
-                wrapper.insertAdjacentHTML('beforeend', markup(isVideo ? "100%" : iframe.height+"px"));
+                console.log(!iframe.height);
+                wrapper.insertAdjacentHTML('beforeend', markup(isVideo || !iframe.height ? "100%" : iframe.height + "px",  isVideo ? "u-position--static" : "u-position--relative"));
```

Comment:
Starting to get pretty bloated here .. the JS is doing too much styling and knowing a little bit too much. But since its not much code atm im sure we can own the debt and refactor if/when it grows with more conditions/styling. Define width & classNames as seperate const instead of inline params. eg. `const width = isVideo || !iframe.height ? "100%" : iframe.height + "px";`